src/web/app/pages-help/instructor-help-page/instructor-help-panel/instructor-help-panel.component.scss

```diff
@@ -1,7 +1,27 @@
 .card-header {
   cursor: pointer;
+
+  .fa-anchor {
+    display: none;
+    opacity: 30%;
+  }
+
+  &:hover {
+    .fa-anchor {
+      display: block;
+
+      &:hover {
+        opacity: 100%;
+      }
+    }
+  }
 }
 
 .card-body {
   overflow: hidden;
 }
+
+.card-header-btn-toolbar {
+  display: flex;
+  gap: 10px;
+}
```

Comment:
In general it's good not to implicitly extend globally defined styles. Doing so makes it confusing for others who want to use the class for another component in the same file.

In this case, I don't think we need another class for this. Let's try to reuse bootstrap's utility css classes. Look into [flex](https://getbootstrap.com/docs/4.6/utilities/flex/) as well as [spacing](https://getbootstrap.com/docs/4.6/utilities/spacing/) classes.